users/README.md

```diff
@@ -34,6 +34,17 @@ default values.
 | MF_USERS_SERVER_KEY       | Path to server key in pem format                                        |                |
 | MF_USERS_SECRET           | String used for signing tokens                                          | users          |
 | MF_JAEGER_URL             | Jaeger server URL                                                       | localhost:6831 |
+| MF_EMAIL_DRIVER           | Mail server driver, mail server for sending reset password token        | smtp           |
```

Comment:
Email server...

Reply:
Actually, mail server is mail server, not email server: https://en.wikipedia.org/wiki/List_of_mail_server_software

So please keep `mail` here.

Reply:
right